language-support/scala/bindings/src/main/scala/com/digitalasset/ledger/client/binding/binding/Primitive.scala

```diff
@@ -47,8 +49,16 @@ sealed abstract class Primitive {
   type Optional[+A] = scala.Option[A]
   val Optional: scala.Option.type = scala.Option
 
-  type Map[+A] = imm.Map[String, A]
-  val Map: imm.Map.type = imm.Map
+  type TextMap[+V] = imm.Map[String, V]
+  val TextMap: imm.Map.type = imm.Map
+
+  @Deprecated // Use TextMap
```

Comment:
```suggestion
  @deprecated("Use TextMap", since = "0.13.38")
```

And likewise for the latter; never use `Deprecated` in scala.

Reply:
fixed